### ISSUE TITLE
Critical dependency: the request of a dependency is an expression

### ISSUE THREAD

**Issue by hoantran-it** [cookpete/react-player#1755]
The latest update introduces this issue
```
WARNING in ./node_modules/react-player/lib/players/Mux.js 136:14-64
Critical dependency: the request of a dependency is an expression
```

Could we disable it?

**Comment by hemantgogia09** [comment]
seeing the same thing with 2.15.0

**Comment by luwes** [comment]
which framework / build tool are you using?

**Comment by hemantgogia09** [comment]
This is some of the libraries that i think useful to know if you think i missed some important ones

> "react": "18.2.0",
> "react-dom": "^18.2.0",
> 
> "eslint": "^8.56.0",
> "eslint-config-airbnb": "^19.0.4",
> "eslint-plugin-import": "^2.29.1",
> "eslint-plugin-jsx-a11y": "^6.8.0",
> "eslint-plugin-react": "^7.33.2",
> "webpack": "^5.89.0",
> "webpack-cli": "^5.1.4",
> "webpack-dev-server": "^4.15.1"

**Comment by ps250551** [comment]
We are seeing this as well, but only in our github runner
![Screenshot 2024-03-01 at 4 39 10 PM](https://github.com/cookpete/react-player/assets/66317988/1c1f613f-ab40-4ee7-b683-51a1a7a0e3c0)

**Comment by hoantran-it** [comment]
> which framework / build tool are you using?

I'm using create-react-app

**Comment by luwes** [comment]
should be fixed in v2.15.1. let me know otherwise!

**Comment by hoantran-it** [comment]
> should be fixed in v2.15.1. let me know otherwise!

awesome, thanks @luwes 👍
